src/main/webapp/WEB-INF/jsp/admin/include/header.jsp

```diff
@@ -74,7 +74,25 @@
     <!-- Brand Logo -->
     <a href="" class="brand-link">
       <img src="https://via.placeholder.com/30" alt="AdminLTE Logo" class="brand-image img-circle elevation-3" style="opacity: .8">
-      <span class="brand-text font-weight-light">AdminLTE 3</span>
+      <span class="brand-text font-weight-light">
+          <c:choose>
+                <c:when test="${sessionScope.loggedIn != null && sessionScope.loggedIn == true}">
```

Comment:
cái này nhầm to rồi, admin với user có 2 trang riêng, mặc dù thiết kế giống nhau

Reply:
đây là trang của admin nên chẳng cần choose when gì hết, trang user cũng chẳng cần luôn, chỉ thay đổi cái chữ thôi

Reply:
ok ô, tui ko để ý, sửa lại